CodeU sp19 Team 27-e0407e25a34b.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This JSON file should not be uploaded to the repo, as it should be kept private, just like a password. Please consider removing it from the pull request.